yarn.lock

```diff
@@ -2,961 +2,890 @@
 # yarn lockfile v1
```

Comment:
You have updated all npm package versions: don't do that (how did you do that?).

In this PR, only bootstrap package should be updated because of examples/Bootstrap